java/server/src/org/openqa/selenium/grid/log/LoggingOptions.java

```diff
@@ -171,4 +238,13 @@ private OutputStream getOutputStream() {
         })
         .orElse(System.out);
   }
+
+  private String getJSONString(Map<String, Object> map) {
+    StringBuilder text = new StringBuilder();
+    try (JsonOutput json = JSON.newOutput(text).setPrettyPrint(false)) {
+      json.write(map);
+      text.append('\n');
```

Comment:
:eyes: Uhhh... Why? The logger will output everything on one line for us.

Reply:
If I passed the Json String as it is, it was pretty printing the Json, so this was added to disable the pretty printing and have the event log in one line.